java/README.md

```diff
@@ -1,85 +1,65 @@
+This preview release is recommended for testing purposes only.
+
 # Getting Started - Java Wrapper
 
-## Notice: Java Wrapper - Work in Progress
+## System Requirements
+
+The beta release of GLIDE for Valkey was tested on Intel x86_64 using Ubuntu 22.04.1, Amazon Linux 2023 (AL2023), and macOS 12.7.
 
-We're excited to share that the Java client is currently in development! However, it's important to note that this client
-is a work in progress and is not yet complete or fully tested. Your contributions and feedback are highly encouraged as
-we work towards refining and improving this implementation. Thank you for your interest and understanding as we continue
-to develop this Java wrapper.
+## Supported Operating Systems
+
+GLIDE for Valkey is supported in Ubuntu, CentOS, and MacOS.
 
 The Java client contains the following parts:
 
-1. `client`: A Java-wrapper around the rust-core client.
-2. `examples`: An examples app to test the client against a Redis localhost
-3. `benchmark`: A dedicated benchmarking tool designed to evaluate and compare the performance of GLIDE for Redis and other Java clients.
-4. `integTest`: An integration test sub-project for API and E2E testing
+1. `src`: Rust dynamic library FFI to integrate with [GLIDE core library](https://github.com/aws/glide-for-redis/blob/main/glide-core/README.md).
+2. `client`: A Java-wrapper around the GLIDE core rust library and unit tests for it.
+3. `examples`: An examples app to test the client against a Valkey localhost.
+4. `benchmark`: A dedicated benchmarking tool designed to evaluate and compare the performance of GLIDE for Valkey and other Java clients.
+5. `integTest`: An integration test sub-project for API and E2E testing.
 
 ## Installation and Setup
 
 ### Install from Gradle
 
 At the moment, the Java client must be built from source.
 
-### Build from source
-
-Software Dependencies:
+#### Prerequisites
 
-- JDK 11+
-- git
-- protoc (protobuf compiler)
-- Rust
+**Protoc installation**
 
-#### Prerequisites
+Download a binary matching your system from the [official release page](https://github.com/protocolbuffers/protobuf/releases) and make it accessible in your $PATH by moving it or creating a symlink.
+For example, on Linux you can copy it to `/usr/bin`:
 
-**Dependencies installation for Ubuntu**
 ```bash
-sudo apt update -y
-sudo apt install -y protobuf-compiler openjdk-11-jdk openssl gcc
-curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
-source "$HOME/.cargo/env"
+sudo cp protoc /usr/bin/
 ```
 
-**Dependencies for MacOS**
-
-Ensure that you have a minimum Java version of JDK 11 installed on your system:
-```bash
- $ echo $JAVA_HOME
-/Library/Java/JavaVirtualMachines/adoptopenjdk-11.jdk/Contents/Home
+Refer to Java's [DEVELOPER.md](https://github.com/aws/glide-for-redis/blob/main/java/DEVELOPER.md) for further instruction on how to set up your development environment.
```

Comment:
Consider using a relative path here for the link.

Reply:
To explain in more detail, replace the following path:

`https://github.com/aws/glide-for-redis/blob/main/java/DEVELOPER.md`

with:

`./DEVELOPER.md`

If the path is specified relative to the markdown file it appears in, the navigation will work but will link to the version of the referenced file in the same branch. This will also work locally in an IDE such as VSCode. An absolute URL will always link to the actual GitHub `main` branch version of the file.